src/include/souffle/datastructure/Brie.h

```diff
@@ -62,7 +62,7 @@ class Trie;
 
 namespace detail::brie {
 
-// FIXME: These data structs should be parameterised/made agonistic `RamDomain`.
+// FIXME: These data structs should be parameterised/made agnoistic `RamDomain`.
```

Comment:
Getting closer, just one `i` to go. It might be clearer with a couple more words, too.
```suggestion
// FIXME: These data structs should be parameterised/made agnostic to `RamDomain` type.
```